webpay/base/dev_messages.py

```diff
@@ -120,9 +127,9 @@ def __init__(self, msg_code):
 
 def _build_legend():
     _legend = {
-        BAD_BANGO_CODE:
-            _('Mozilla received an invalid code from the payment '
-              'provider (Bango) when processing the payment'),
+        BAD_BANGO_CODE: _(
```

Comment:
This message doesn't match the formatting of the other ones now, if that matters.